src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,20 @@
+.stars {
+  display: flex;
+
+  &__star {
+    margin-right: 4px;
+    width: 16px;
+    height: 16px;
+    background-repeat: no-repeat;
+    background-position: center;
+    background-image: url(../images/star.svg);
+  }
+
+  &:last-child {
+    margin-right: 0;
+  }
+
+  &__star:nth-child(-n + 4) {
+    background-image: url(../images/star-active.svg);
+  }
```

Comment:
use the whole `stars` block to make it possible to set any amount of stars

Reply:
Please advise I need to use this? - 
.stars--1 :nth-child(-n + 1),
.stars--2 :nth-child(-n + 2),
.stars--3 :nth-child(-n + 3),
.stars--4 :nth-child(-n + 4),
.stars--5 :nth-child(-n + 5) {
  background-image: url(images/star-active.svg);
}

and concerning margin for card name, yesterday was Q/A and your colleague advised to use margin: 40px 0 4px; tests without it doesn't work 
![Screenshot 2024-05-09 at 10 42 27](https://github.com/mate-academy/layout_product-cards/assets/155059435/e32a13a5-0fd7-444f-a069-7e21dddc3831)

Reply:
We can add top margin to the product code. Doesn't it work?